### ISSUE TITLE
YSDA frequently asked questions

### ISSUE THREAD

**Issue by justheuristic** [probabll/dgm4nlp#1]
Thanks for the great notebook! Most of the students were able to get through it successfully. There were, however, some common problems that 

seminar 1 frequently asked questions:
* a lot of students complained about not being able to test their code right away.
It might be a good idea to add small code snippets that test each part of the model. Here's an [example](https://github.com/yandexdataschool/nlp_course/blob/master/week03_lm/seminar.ipynb) by Lena
* what is the shape of probs/logits param in bernoulli? ([B])
* are probs probabilities of 0 or 1? (1)
* what is output shape of bernoulli KL? [B]?
* a lot of people got confused with bag-of-words encoder: 
  * it wasn't clear what kind of "output dimension" is there. It might be a good idea to clarify that for bag of words case O = I
  * many student asked questions on what VALUES are in the mask. (1's for words that are used, 0's for what is omitted)
